### ISSUE TITLE
Unable to connect web socket,click to reconnect 

### ISSUE THREAD

**Issue by Acsriot94** [Ylianst/MeshCentral#5612]
**Describe the bug**

**To Reproduce**
1. Start Mesh Central
2. Attempt To Login
3. Get "Unable to connect to Web Socket"

**Expected behavior**
Logs into the Control GUI

**Screenshots**

**Server Software (please complete the following information):**
 - OS: Truenas SCALE
 - Virtualization: Kubernauts
 - Network: LAN, Reverse Proxy

**Client Device (please complete the following information):**
 - Device: Macbook M1 Pro, Windows 11 Workstation
 - OS: Windows, Mac
 - Network: LAN + Wan
 - Browser: Have attempted Firefox, Google 
 - MeshCentralRouter Version: N/A

**Remote Device (please complete the following information):**
 - Device: [e.g. Laptop]
 - OS: [e.g. Windows 10 21H2]
 - Network: [e.g. Local to Meshcentral, Remote over WAN]
 - Current Core Version (if known): [**HINT**: Go to a device then `console` Tab then type `info`]

** I am still figuring out how to generate this
**Your config.json file**

[meshcentral_meshcentral-mongodb-7b5dc76996-6hbv6_meshcentral-mongodb.log](https://github.com/Ylianst/MeshCentral/files/13559206/meshcentral_meshcentral-mongodb-7b5dc76996-6hbv6_meshcentral-mongodb.log)
[meshcentral_meshcentral-785bdb9f7b-lwtjb_meshcentral.log](https://github.com/Ylianst/MeshCentral/files/13559207/meshcentral_meshcentral-785bdb9f7b-lwtjb_meshcentral.log)

**Comment by NiceGuyIT** [comment]
You purposefully removed the bug report template that asks for information needed to troubleshoot. Why?

**Comment by hansolo199532** [comment]
Updated, sorry very new to Github!

**Comment by NiceGuyIT** [comment]
Your meshcentral log doesn't show any connections. Your mongodb log has the following error: `Client sent an HTTP request over a native MongoDB connection`. It would seem that external connections are trying to connect to MongoDB, not MeshCentral.

> 2023-12-04T22:17:08.982917771Z {"t":{"$date":"2023-12-04T22:17:08.982+00:00"},"s":"I",  "c":"NETWORK",  "id":22988,   "ctx":"conn90","msg":"Error receiving request from client. Ending connection from remote","attr":{"error":{"code":17,"codeName":"ProtocolError","errmsg":"**Client sent an HTTP request over a native MongoDB connection**"},"remote":"172.16.0.1:53478","connectionId":90}}


Note: You should not put your logs in a code block. How you had the logs before was good. I.e. remove the code block but keep the links.

**Comment by hansolo199532** [comment]
What would that have somting to do with Reverse proxy?

**Comment by NiceGuyIT** [comment]
Yes. HTTP requests are going to Mongo, not the application. Look at your reverse proxy configuration. Verify your ports are pointing to the right service.

**Comment by KellyDi0te** [comment]
I have the same error when I use NGINX reverse-proxy : 
Let's use the following DNS names : 
- NGINX : ip-meshcentral.mydomain
- MeshCentral Server 1 : ip-meshcentral1.mydomain
- MeshCentral Server 2 : ip-meshcentral2.mydomain
- MeshCentral Server 3 : ip-meshcentral3.mydomain

I just finished yesterday the configuration of load-balancing.
Now, when I try to use dns-meshcentral.mydomain to be redirected to 1 of the 3 servers, I have the error "Unable to connect to Web Socket, click to reconnect"
When I go directly into one of the 3 servers (e.g : dns-meshcentral1.mydomain), I have no problem to connect. 

I hope this gives you an idea of where to go and what to do.

**Comment by si458** [comment]
@KellyDi0te you need to share your nginx config and also your meshcentral config otherwise how can we help?

**Comment by hansolo199532** [comment]
is there any direction y'all can point me in ? I don't see any place to configure Traefik with MeshCentral
